weldx/visualization/colors.py

```diff
@@ -192,8 +192,7 @@ def color_generator_function() -> Generator:
 
     """
     while True:
-        for color in _color_list:
-            yield color
+        yield from _color_list
```

Comment:
What I really love about such tools is that they also teach you how to improve your coding style 😃 Didn't know this synthax